src/theme.ts

```diff
@@ -252,6 +255,7 @@ export const newTheme = extendTheme({
         },
         text: {
           primary: White, // Various MUI components and text
+          secondary: White,
```

Comment:
Is it OK for primary and secondary to be identical?

Reply:
I don't remember what that was for, but I notice now it makes a few unnecessary changes-I'll remove it